src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -0,0 +1,161 @@
+      padding-left: var(--pf-c-clipboard-copy__group-copy--PaddingLeft);
+      border: none;
+
+      &::before {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        content: "";
+        border: var(--pf-c-clipboard-copy__group-copy--BorderWidth) solid;
+        border-color: var(--pf-c-clipboard-copy__group-copy--BorderTopColor) var(--pf-c-clipboard-copy__group-copy--BorderRightColor) var(--pf-c-clipboard-copy__group-copy--BorderBottomColor) var(--pf-c-clipboard-copy__group-copy--BorderLeftColor);
+      }
+
+      &:hover,
+      .pf-m-hover {
+        &::before {
+          --pf-c-clipboard-copy__group-copy--BorderBottomColor: var(--pf-c-clipboard-copy__group-copy--hover--BorderBottomColor);
+        }
+      }
+    }
+
+    .pf-c-clipboard-copy__group-text {
+      position: relative;
+      max-width: var(--pf-c-clipboard-copy__group-text--MaxWidth);
+      padding: var(--pf-c-clipboard-copy__group-text--PaddingTop) var(--pf-c-clipboard-copy__group-text--PaddingRight) var(--pf-c-clipboard-copy__group-text--PaddingTop) var(--pf-c-clipboard-copy__group-text--PaddingLeft);
+      overflow: hidden;
+      font-size: var(--pf-c-clipboard-copy__group-text--FontSize);
+      line-height: var(--pf-c-clipboard-copy__group-text--LineHeight);
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      background-color: var(--pf-c-clipboard-copy__group-text--BackgroundColor);
+      border: var(--pf-c-clipboard-copy__group-text--BorderWidth) solid;
+      border-color: var(--pf-c-clipboard-copy__group-text--BorderTopColor) var(--pf-c-clipboard-copy__group-text--BorderRightColor) var(--pf-c-clipboard-copy__group-text--BorderBottomColor) var(--pf-c-clipboard-copy__group-text--BorderLeftColor);
+    }
+  }
+  /* stylelint-enable */
+
+  .pf-c-clipboard-copy__expanded-content {
+    position: absolute;
+    top: var(--pf-c-clipboard-copy__expanded-content--Top);
+    z-index: var(--pf-c-clipboard-copy__expanded-content--ZIndex);
+    max-width: 100%;
+    padding: var(--pf-c-clipboard-copy__expanded-content--PaddingTop) var(--pf-c-clipboard-copy__expanded-content--PaddingRight) var(--pf-c-clipboard-copy__expanded-content--PaddingBottom) var(--pf-c-clipboard-copy__expanded-content--PaddingLeft);
+    background-color: var(--pf-c-clipboard-copy__expanded-content--BackgroundColor);
+    background-clip: padding-box;
+    border: var(--pf-c-clipboard-copy__expanded-content--BorderWidth) solid transparent;
+    box-shadow: var(--pf-c-clipboard-copy__expanded-content--BoxShadow);
+  }
+
+  // Toggle button expanded
+  &.pf-m-expanded .pf-c-clipboard-copy__group-toggle {
+    &::before {
+      --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--expanded--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-clipboard-copy__group-toggle--expanded--BorderBottomWidth);
+    }
+
+    i {
```

Comment:
this should probably use a BEM class. `i` won't work in react since they use `svg`